cpu/cortexm_common/include/cpu.h

```diff
@@ -51,6 +51,14 @@ extern "C" {
 #define STACK_CANARY_WORD   (0xE7FEE7FEu)
 
 /**
+ * @brief   All Cortex-m-based CPUs provide pm_set_lowest
+ *
+ * The pm_set_lowest is provided either the pm_layered module if used, or
```

Comment:
minor: I think there's a "by" missing in this sentence.

Reply:
jupp. Fixed and squashed